AEPCore/Tests/RulesTests/TokenFinderTests.swift

```diff
@@ -123,9 +124,9 @@ class TokenFinderTests: XCTestCase {
             return
         }
         let date_UNIX = Date(timeIntervalSince1970: TimeInterval(date_UNIX_Int))
-        /// Then: return same timestamp with different format
-        XCTAssertEqual(date_ISO8601, date_ISO8601NoColon)
-        XCTAssertEqual(date_ISO8601, date_UNIX)
+        // Then: return same timestamp with different format, but only compare seconds as ISO8601NoColon and UNIX do not count milliseconds
+        XCTAssertEqual(0, Calendar.current.dateComponents([.second], from: date_ISO8601, to: date_ISO8601NoColon).second)
+        XCTAssertEqual(0, Calendar.current.dateComponents([.second], from: date_ISO8601, to: date_UNIX).second)
```

Comment:
As `date_ISO8601` contains milliseconds, it cannot be directly compared to the dates with only seconds.